src/monopoly.js

```diff
@@ -128,31 +125,35 @@ export default class Monopoly {
         return this;
     }
 
-    findFirstPlayer(){//TODO what if ties occur
+    findFirstPlayer() {//TODO what if ties occur
         this.random.shuffle(this.playerArray);
         this.currentPlayer = 0;
     }
 
-    loadPropertyArray(){
-        const PROPERTY_FUNCTIONS = {
-            'standardProperty': standardProperty,
-            'railroad': railroad,
-            'utility': utility,
-            'eventCard' : eventCard,
-            'noEvent' : noEvent,
-            'go' : go,
-            'goToJail' : goToJail,
-            'incomeTax' : incomeTax,
-            'luxuryTax' : luxuryTax,
+    loadPropertyArray() {
+        this.propertyArray = new Array();
+
+        const PROPERTY_TYPES = {
+            'standardProperty' : StandardProperty,
+            'railroad': Railroad,
+            'utility': Utility,
+            'eventCard' : EventCard,
+            'noEvent' : NoEvent,
+            'go' : Go,
+            'goToJail' : GoToJail,
+            'incomeTax' : IncomeTax,
+            'luxuryTax' : LuxuryTax,
         };
 
-        this.propertyArray = JSON.parse(fs.readFileSync('properties.json'));
-        this.propertyArray.forEach((property) => {
-            property.landOnFunction = PROPERTY_FUNCTIONS[property.functionID];
-        });
+        const JSONArray = JSON.parse(fs.readFileSync('properties.json'));
+        for (let property of JSONArray) {
```

Comment:
I think `property` can be `const` here.  I believe the way for loops work is that it's as if you declared the variable at the start of the inside of the block.  So you're not rebinding it.